### PR TITLE
adding server-status pages

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/roles/nginx/defaults/main.yml [hunk 1]
```diff
@@ -33,8 +33,24 @@ NGINX_LOG_FORMAT_NAME: 'p_combined'
 # headers to reflect the properties of the incoming request.
 NGINX_SET_X_FORWARDED_HEADERS: False
 
+
```
Comment (e0d) [inline playbooks/roles/nginx/defaults/main.yml:36]: Pull edX specifc stuff out

**Comment by e0d** [comment]
After the comment, :+1:

File: playbooks/roles/nginx/defaults/main.yml (hunk 1 above)
Comment (jarv) [inline playbooks/roles/nginx/defaults/main.yml:36]: Everything is generic except for the image, should we just have a placeholder?

Comment (jarv) [inline playbooks/roles/nginx/defaults/main.yml:36]: ok using https://upload.wikimedia.org/wikipedia/commons/thumb/1/11/Pendleton_Sinking_Ship.jpg/640px-Pendleton_Sinking_Ship.jpg as the placeholder..

**Pull request merged**